src/translations/nbTranslations.json

```
@@ -462,19 +463,22 @@
         "category_limitations": "Begrensning for kategori",
         "category_limitations_helper_text": "<p>Kanaleier kan velge om tilgangsbegrensninger skal gjelde for alle resultatkategorier eller bare noen utvalgte.</p><heading>Publiseringsforespørsler</heading><p>For resultatkategorier MED tilgangsbegrensning blir publiseringsforespørsler sendt til filkurator hos kanaleier.</p><p>For resultatkategorier UTEN tilgangsbegrensning, blir publiseringsforespørsler sendt til filkurator ved institusjonen som registrator er logget inn på.</p>",
         "channel_claim": "Kanaleierskap",
-        "channel_claims_overview_description": "Oversikt over kanaler fra Kanalregisteret:",
+        "channel_claims_overview_description": "Kanaler med redaktør",
         "channel_claims_settings_description": "Administrer eierskap og tilganger til kanaler fra Kanalregisteret:",
         "channel_owner": "Kanaleier",
-        "channel_owner_helper_text": "<p>Institusjonen som har tatt eierskap til kanalen (utgiverinstitusjon).</p><p>Hvis en kanal har feil kanaleier, ta kontakt med institusjonen for å få rettet feilen.</p>",
+        "channel_owner_helper_text": "<p>Institusjonen som har tatt det redaksjonelle ansvar for innhold utgitt av kanalen.</p><p>Hvis en kanal har feil redaktør, ta kontakt med institusjonen for å få rettet feilen.</p>",
```

skulle vært bestemt form her?
```suggestion
        "channel_owner_helper_text": "<p>Institusjonen som har tatt det redaksjonelle ansvaret for innholdet utgitt av kanalen.</p><p>Hvis en kanal har feil redaktør, ta kontakt med institusjonen for å få rettet feilen.</p>",
```